cleancode/src/main/java/com/epam/cleancode/comments/InvalidInputException.java

```diff
@@ -1,16 +1,14 @@
 package com.epam.cleancode.comments;
 
 public class InvalidInputException extends RuntimeException{
-    String message;
-    public InvalidInputException(String s) {
+    private String message;
```

Comment:
Throwable already has message text field. Don't need to add new one. please check and remove. Also do not forget about code alignment